pageserver/src/walredo.rs

```diff
@@ -333,7 +334,9 @@ impl PostgresRedoManager {
                 // and hence the current `apply_wal_records()` calls will observe
                 //  `output.stdout.as_raw_fd() != stdout_fd` .
```

Comment:
I don't understand the race condition that the comment talks about.

Reply:
Ok I think I understand it now. The comment could use some copy-editing..

Reply:
I updated the PR description.